tests/storage/test_profile.py

```diff
@@ -64,64 +62,3 @@ def test_avatar_url(self) -> None:
         self.assertIsNone(
             self.get_success(self.store.get_profile_avatar_url(self.u_frank))
         )
-
```

Comment:
The rationale for dropping these tests is that they both were testing background updates that have already run, and the changes to the code in this PR result in making it pretty impossible to run the tests in sqlite. When I added these tests I felt like they were pretty borderline useful (basically only as a proof of concept) so I think it makes sense to drop them now.